obs-packaging/kata-containers-image/debian.rules-template

```diff
@@ -9,8 +9,8 @@ override_dh_auto_install:
 	tar xzf /usr/src/packages/SOURCES/kata-containers.tar.gz -C /usr/src/packages/SOURCES/ ; \
 	image=$$(find /usr/src/packages/SOURCES/ -type f -name '*.img') ; \
 	initrd=$$(find /usr/src/packages/SOURCES/ -type f -name '*.initrd') ; \
-	install -D $${image} ./debian/kata-containers-image/usr/share/kata-containers/ ; \
-	install -D $${initrd} ./debian/kata-containers-image/usr/share/kata-containers/ ; \
+	install -m 0400 -D $${image} ./debian/kata-containers-image/usr/share/kata-containers/ ; \
+	install -m 0400 -D $${initrd} ./debian/kata-containers-image/usr/share/kata-containers/ ; \
```

Comment:
Whilst we're at it, why not add `-o root -g root` too?

Any thoughts on that @marcov?

Reply:
I'd expect `-o root -g root` to be the defaults.

I'm not sure about what are the pros of preventing read for non-root though.